hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis_snapshot/OmRatisSnapshotProvider.java

```diff
@@ -143,8 +144,8 @@ public void downloadSnapshot(String leaderNodeID, File targetFile)
     OMNodeDetails leader = peerNodesMap.get(leaderNodeID);
     URL omCheckpointUrl = leader.getOMDBCheckpointEndpointUrl(
         httpPolicy.isHttpEnabled(), true);
-    LOG.info("Downloading latest checkpoint from Leader OM {}. Checkpoint " +
-        "URL: {}", leaderNodeID, omCheckpointUrl);
+    LOG.info("Downloading latest checkpoint from Leader OM {}. Checkpoint: {} " + "URL: {}",
```

Comment:
nit: concatenation is unnecessary.

```suggestion
    LOG.info("Downloading latest checkpoint from Leader OM {}. Checkpoint: {} URL: {}",
```